README.md

```diff
@@ -66,24 +66,29 @@ Astarte's Documentation.
 
 ## Kubernetes support
 
-| Kubernetes Version | Supported | Tested by CI |
-| --- | --- | --- |
-| v1.17.x  | :x: | :x: |
-| v1.18.x  | :x: | :x: |
-| v1.19.x  | :white_check_mark: | :white_check_mark: |
-| v1.20.x  | :white_check_mark: | :white_check_mark: |
-| v1.21.x  | :white_check_mark: | :white_check_mark: |
+| Kubernetes Version | Supported              | Tested by CI           |
+| ---                | ---                    | ---                    |
+| v1.18.x            | :x:                    | :x:                    |
+| v1.19.x            | :large_orange_diamond: | :large_orange_diamond: |
+| v1.20.x            | :white_check_mark:     | :white_check_mark:     |
+| v1.21.x            | :white_check_mark:     | :white_check_mark:     |
+| v1.22.x            | :white_check_mark: \*  | :white_check_mark: \*  |
+| v1.23.x            | :white_check_mark: \*  | :white_check_mark: \*  |
```

Comment:
What if we use the `*` for releases still supporting AVI instead? We switched to ADI because k8s deprecated the networking API on which it was based

Reply:
++, otherwise we'd have to keep `*` forever, while it should only be for special cases

Reply:
Also, didn't we agree on using a different symbol to make sure users check the footnotes? Something like :warning: